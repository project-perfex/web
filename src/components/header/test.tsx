import { render, screen } from '@testing-library/react'

import { Header } from '.'

describe('<Main />', () => {
  it('should render the heading', () => {
    const { container } = render(<Header />)

    expect(screen.getByRole('heading', { name: /header/i })).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
