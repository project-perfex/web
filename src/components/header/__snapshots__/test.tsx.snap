// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<header>
  <h1>
    Header
  </h1>
</header>
`;
